recipes/dbg-macro/all/conanfile.py

```diff
@@ -13,31 +15,25 @@ class DbgMacroConan(ConanFile):
     settings = ("compiler", )
     no_copy_source = True
 
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
 
     def source(self):
-        tools.get(**self.conan_data["sources"][self.version])
-        extracted_dir = self.name + "-" + self.version
-        os.rename(extracted_dir, self._source_subfolder)
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
 
-    def configure(self):
-        minimal_cpp_standard = "11"
-        if self.settings.get_safe("compiler.cppstd"):
-            tools.check_min_cppstd(self, minimal_cpp_standard)
+    def validate(self):
+        check_min_cppstd(self, 17)
```

Comment:
We need to have an extra check this, please check https://github.com/conan-io/conan-center-index/blob/master/docs/package_templates/header_only/all/conanfile.py#L67, it should be straightforward to copy

Reply:
Didn't copy and paste it, but made it similarly.

I can't see the point of creating the class property for such simple thing.
I hope you're ok with my attempt.